tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -74,6 +74,21 @@ export const getActiveRunId = createSelector(
   (state: DebuggerState): string | null => state.activeRunId
 );
 
+/**
+ * Selectors related to data polling.
+ */
+
+/**
+ * Get the time elapsed from the last time at which a data polling
+ * yielded new data and the last polling time.
+ */
+export const getPollSilenceTime = createSelector(
```

Comment:
`PollSilence` is this a common phrase? It is quite foreign to me and I still don't understand it reading the comment above. 

Also, `lastNewPollDataTime` does not make much sense to me either. Can we do better property/variable naming here?

~Perhaps something like `getDebugTimeSinceMeaningfulData`? (I hope you can change `lastNewPollDataTime` to something like `lastDatafulLoadedTimeInMs`~

I think this selector should change to `getDebugDurationBetweenPollAndNonEmptyDataFetch` (FYI, `getDebug` prefix is important since all selectors ideally should be re-exported at the root level and it avoids collisions).

Reply:
Also, please specify the unit (of time) throughout this PR. Without it, code is much less readable.

Reply:
I'd argue that poll silence is a nice and short name for the slightly involved context here. It is how long polling has given empty (silent) responses. The name you suggested is precise but a mouthful and hence hard to type, think and say. I initially wrote a function name similar to it but decided against it. I personally believe that sometimes inventing new terminology helps understanding.

Units are added to the new fields and functions in this PR.